<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSelectIcon</class>
 <widget class="QDialog" name="DialogSelectIcon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>99</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>import icon resource</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_qtv">
       <property name="text">
        <string>Filename</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_QTV_filename"/>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_QTV_browser">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_QTV_4">
       <property name="text">
        <string>centerx</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_QTV_centx"/>
     </item>
     <item>
      <widget class="QLabel" name="label_QTV_3">
       <property name="text">
        <string>centery</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_QTV_centy"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_QTV_2">
       <property name="text">
        <string>name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_QTV_name"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_QTV_ok">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_QTV_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
